src/ui-components/core-views.tsx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+
+import HeightAndWeightOverview from "../widgets/heightandweight/heightandweight-overview.component";
+import VitalsOverview from "../widgets/vitals/vitals-overview.component";
+import ConditionsOverview from "../widgets/conditions/conditions-overview.component";
+import AllergyOverview from "../widgets/allergies/allergy-overview.component";
+import NotesOverview from "../widgets/notes/notes-overview.component";
+import ProgramsOverview from "../widgets/programs/programs-overview.component";
+import MedicationsOverview from "../widgets/medications/medications-overview.component";
+import { DashboardConfigType } from "./dashboard/dashboard.component";
+
+const coreMultiDashboards = {};
+export const coreDashboards: CoreDashboardsType = {
+  summaryDashboard: {
+    name: "summaryDashboard",
+    title: "Overview",
+    layout: {
+      columns: 4
+    },
+    widgets: [
+      {
+        name: "conditionsOverview",
+        layout: { columnSpan: 2 }
```

Comment:
It's nice that the new config allows you to specify the layout. We are really getting there with the configurable patient chart.